<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>equivalencias</class>
 <widget class="QDialog" name="equivalencias" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edición / Alta de equivalencias</string>
  </property>
  <layout class="QVBoxLayout" name="_2" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="DescripcionLineEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Cuenta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentaLineEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength" >
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buscapushButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-55.png</normaloff>:/images/librok-55.png</iconset>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox" >
       <property name="text" >
        <string>Ignorar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="BotonGuardar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Guardar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-4.png</normaloff>:/images/librok-4.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonEliminar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Eliminar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-17.png</normaloff>:/images/librok-17.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCancelar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-7.png</normaloff>:/images/librok-7.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonSalir" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tabla" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>BotonSalir</sender>
   <signal>clicked()</signal>
   <receiver>equivalencias</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>372</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>420</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
